src/components/Collections/CollectionOptions/QuickOptions/MoveToTrashOption.tsx

```diff
@@ -0,0 +1,26 @@
+import { CollectionActions } from '..';
+import React from 'react';
+import { IconButton, Tooltip } from '@mui/material';
+import DeleteOutlinedIcon from '@mui/icons-material/DeleteOutlined';
+import constants from 'utils/strings/constants';
+
+interface Iprops {
+    handleCollectionAction: (
+        action: CollectionActions,
+        loader?: boolean
+    ) => (...args: any[]) => Promise<void>;
+}
+
+export function MoveToTrashOption({ handleCollectionAction }: Iprops) {
```

Comment:
Let's name it `EmptyTrashQuickOption` ?  as it is doing that... `emptyTrash` and `moveToTrash` are different things with different logic/ API ... so this naming can cause confusion

Reply:
this is not done yet no?, the component is still called `moveToTrashOption` ?